ci-operator/step-registry/operatorhub/subscribe/metallb-operator/operatorhub-subscribe-metallb-operator-commands.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -z "${METALLB_OPERATOR_SUB_INSTALL_NAMESPACE}" ]]; then
+  echo "ERROR: INSTALL_NAMESPACE is not defined"
+  exit 1
+fi
+
+if [[ -z "${METALLB_OPERATOR_SUB_PACKAGE}" ]]; then
+  echo "ERROR: PACKAGE is not defined"
+  exit 1
+fi
+
+if [[ -z "${METALLB_OPERATOR_SUB_CHANNEL}" ]]; then
+  echo "ERROR: CHANNEL is not defined"
+  exit 1
+fi
+
+echo "Installing ${METALLB_OPERATOR_SUB_PACKAGE} from ${METALLB_OPERATOR_SUB_CHANNEL} into ${METALLB_OPERATOR_SUB_INSTALL_NAMESPACE}"
+
+# create the install namespace
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "${METALLB_OPERATOR_SUB_INSTALL_NAMESPACE}"
+  labels:
+    openshift.io/cluster-monitoring: "true"
+EOF
+
+# deploy new operator group
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: "${METALLB_OPERATOR_SUB_INSTALL_NAMESPACE}"
+  namespace: "${METALLB_OPERATOR_SUB_INSTALL_NAMESPACE}"
+spec: {}
+EOF
+
+# subscribe to the operator
+cat <<EOF | oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: "${METALLB_OPERATOR_SUB_PACKAGE}"
+  namespace: "${METALLB_OPERATOR_SUB_INSTALL_NAMESPACE}"
+spec:
+  channel: "${METALLB_OPERATOR_SUB_CHANNEL}"
+  installPlanApproval: Automatic
+  name: "${METALLB_OPERATOR_SUB_PACKAGE}"
+  source: "${METALLB_OPERATOR_SUB_SOURCE}"
+  sourceNamespace: openshift-marketplace
+EOF
+
+# can't wait before the resource exists. Need to sleep a bit before start watching
+sleep 60
+
+RETRIES=30
+CSV=
+for i in $(seq "${RETRIES}"); do
+  if [[ -z "${CSV}" ]]; then
```

Comment:
If i'm not wrong, we might be able to change all the double square brackets to single.

`s/[[/[/;s/]]/]/`.

Reply:
Could you tell me why should I use double square brackets instead of single, since is works to use single here